pkg/bundle/sync.go

```diff
@@ -131,6 +132,12 @@ func (b *bundle) buildSourceBundle(ctx context.Context, bundle *trustapi.Bundle)
 
 	resolvedBundle.data = strings.Join(bundles, "\n") + "\n"
 
+	resolvedBundle.binaryData = make(map[string][]byte)
+	err := populateAdditionalFormatData(resolvedBundle.data, bundle.Spec.Target, resolvedBundle.binaryData)
+	if err != nil {
+		return bundleData{}, fmt.Errorf("failed encoding additional formats: %w", err)
+	}
```

Comment:
Yeah, I didn't want to change too much to make it easier for https://github.com/cert-manager/trust-manager/pull/244. That was also noted in the PR description. But I'll take a look and see if this can be slightly more refactored now.

Reply:
Ok, I refactored some more. Hope this is closer to what you had in mind. PTAL!

Reply:
Oh totally my bad - I missed that in the description. I would've totally been fine with you skipping this with that in mind! Thank you for doing it though 😁